Diagnostics/FreeBusyChecker/Functions/hostOutput.ps1

```diff
@@ -0,0 +1,130 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '', Justification = 'Variables are being used in functions')]
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory = $false)]
+    $TestingDAuthConfiguration = "---------------------------------------Testing DAuth Configuration-----------------------------------------------",
+    $TestingOAuthConfiguration = " ---------------------------------------Testing OAuth Configuration----------------------------------------------- ",
+    $CollectingExoAvailabilityInformation = " Collecting Exchange Online Availability Information",
+    $ExchangeOnlinePowershellModuleMessage = "`n Exchange Online Powershell Module is required to Check Free Busy Configuration on Exchange Online side. Installing Module",
+    $ExchangeOnlineModuleAvailableMessage = "`n ExchangeOnlineManagement module is available.",
+    $TestingExoDAuthConfiguration = " ---------------------------------------Testing DAuth Configuration----------------------------------------------- ",
+    $TestingExoOAuthConfiguration = " ---------------------------------------Testing OAuth Configuration----------------------------------------------- ",
+    $ThatIsAllForTheExchangeOnlineSide = " That is all for the Exchange Online Side"
+)

```

Comment:
Not sure what is going on here. We started a new file and started it off like a completely new script with `[CmdletBinding()]` on the top with `param` This should only need to be done for the main script and new `Function`(s)

Reply:
Mistake, removed